.github/actions/check-branch-commits/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: "Check Branch Commits"
+description: |
+  Runs the branch commit checker on aptos-core, assuming it is already checked out in the current working directory.
+inputs:
+  TIMEOUT_MINUTES:
+    description: "The number of minutes to wait for branch commit checking to finish."
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Install Dependencies
+      shell: bash
+      run: |
+        echo "${HOME}/bin/" >> $GITHUB_PATH # default INSTALL_DIR to path
+        sudo apt-get update -y && sudo apt-get install -y expect
```

Comment:
is the `expect` package actually used anywherE? I don't see it being used.

Reply:
Copy/pasta from all the other jobs. Removed here (and from `fullnode-sync`).